draft-ietf-tls-rfc4492bis.xml

```diff
@@ -741,7 +733,8 @@
         <t> All RSA signatures must be generated and verified according to <xref target="PKCS1"/> block type 1.</t>
       </section>
       <section anchor="valid25519" title="Public Key Validation">
-        <t> With the NIST curves, each party must validate the public key sent by its peer. A receiving party MUST 
+        <t> With the NIST curves, each party must validate the public key sent by its peer in the ClientKeyExchange
```

Comment:
There's a MUST in the next sentence, which is more technical. Do we need one here as well?